raw_data/preprocessed/africa/sierra_leone_english.txt

```diff
@@ -0,0 +1,118 @@
+
+Functions of the Board.
+(4) The Board shall: [a] facilitate coordination, cooperation and collaboration among Government Ministries, local authorities and other governmental agencies in all areas relating to environmental protection; [b] review national and sectoral environmental policies and make such recommendations or proposals it may think necessary to the Minister; [c] review environmental impact assessments prepared pursuant to this Act and make appropriate recommendations to the Director; [d] investigate or cause to be investigated any activity, occurrence or transaction which it considers is likely to have or result in harmful consequences to the environment and advise on measures necessary to prevent or minimise such consequences; [e] advise the Minister on areas of environmental protection and control requiring special or additional measures, indicating the priorities and specific goals to be achieved; [f] undertake or cause to be undertaken specific studies and research aimed at developing strategies for the protection of the environment and make appropriate recommendations to the Minister; and [g] consider any other matters which may be referred to it by the Minister and make appropriate recommendations or proposals1hI::moo.. [s] [I] Subjectto theprovisioDsofIbis seclim, theBc.dshall Mrs;".of meet at suchtimes and places as it thinks expedientfortbe traIJSaCIion dieBoad.. of its business. [2] Special meetings of theBaud may be CODveued at the request of the Minister. [3] The quorum for a meetingm1heBoanI shaD be seven. [4] Tbecbairmanshall presideatmeehDSSof tbeBoaRIaod in his absence a member of the 80aRI appointed by die membem present shall preside. 6 No.2 Environment Protection Act & [5] The Board may co-opt any person to advise it on any issue at its meetings and such co-opted person may participate in the deliberations of the Boardbutshall not voteonany matterfor decision bythe Board. [6] Decisions at meetingsof the Board shall be taken by a majority of thememberspresentand votingandthechairman or other person presiding shall, in the eventof equalityin votes, have a casting v~. . Committees. [5] The Board may establish and constitute such committees thereoftoperform such functions asit may thinknecessary. Pr~ure d [1] Subjecttotheprovisionsof thisAct,theBoardcudcommittees ~a:.. thereof shall regulate the procedure at their meetings. Honorarium. [8] Members of the Boardand committees thereof s~ll be paid suchhonorarium as theMinisterafterconsultation withtheMnisterfor Finance may determine.
```

Comment:
It looks like articles 5 through 8 have been added to article 4. There are now a number of errors in this article, as you can see. Please address.